src/commands/api/generate.ts

```diff
@@ -0,0 +1,22 @@
+import { spawn } from 'node:child_process';
+
+import { Command } from '@oclif/core';
+
+import logger from 'utils/logger';
+
+export default class ApiGenerate extends Command {
+  // TODO: yarn add @mondaycom/setup-api - when this package is published
+  static description = 'Run the setup-api-monday command from @mondaycom/setup-api';
```

Comment:
align description to what is in the Readme and remove additions from the Readme (which were done manually)